js/pillbox.js

```diff
@@ -35,7 +37,15 @@
 
 	// PILLBOX CONSTRUCTOR AND PROTOTYPE
 
-	var Pillbox = function (element, options) {
+	var ENTER_KEYCODE = 13;
+	var COMMA_KEYCODE = 188;// `,` & `<` unless you check for shift modifier key not being pressed on event object (we don't)
```

Comment:
either ALL_UPPER_CONST or isFunctions for keycode checks. which way should we go?